trollflow2/dict_tools.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019 Pytroll developers
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+# Workaround for unittests so that satpy and posttroll installations
+# are not necessary
+"""Tools for product list operations."""
+
+import dpath
+
+def plist_iter(product_list, base_mda=None, level=None):
```

Comment:
Some explanation on the workings of `plist_iter()` would be nice. Docstring at the least.

Reply:
done